pkg/trace/test/agent.go

```diff
@@ -41,6 +41,9 @@ func newAgentRunner(ddAddr string, verbose bool) (*agentRunner, error) {
 		if verbose {
 			log.Print("agent: trace-agent not found, trying to install...")
 		}
+		// TODO(x): Do not use $GOBIN/trace-agent. Install a new one every time.
```

Comment:
Isn't this addressed in https://github.com/DataDog/datadog-agent/pull/6906 ?

Reply:
True. Still need to merge that.